packages/backend/src/tenants/service.ts

```diff
@@ -186,3 +191,23 @@ async function deleteTenant(
     throw err
   }
 }
+
+async function updateOperatorApiSecretFromConfig(
+  deps: ServiceDependencies
+): Promise<undefined | TenantError> {
+  const { adminApiSecret, operatorTenantId } = deps.config
+
+  const tenant = await Tenant.query(deps.knex)
+    .findById(operatorTenantId)
+    .whereNull('deletedAt')
+    .first()
+
+  if (!tenant) {
+    return TenantError.TenantNotFound
+  }
+  if (tenant.apiSecret !== adminApiSecret) {
+    await Tenant.query(deps.knex)
+      .patch({ apiSecret: adminApiSecret })
+      .where('id', operatorTenantId)
+  }
+}
```

Comment:
New method to update the secret from config. I made a new method rather than using the update because:

- I wanted to be opinionated about where this is coming from definitely the config, not just some value being passed in which is intended to be from the config.
- I didnt want to update unless its changed. didnt want to add that logic to the update just for this. Perhaps we could just always write it but figured this was safer.

Reply:
I was just noticing that this exemplifies the value of the repo layer... notice we use the cache in the service `get` method but we cant use that here (could if we were getting from the repo). I suppose if thats important I can use/update the cache here as well.

Reply:
Makes sense to have another method. As far as caching goes, I think we should update it if we end up updating the secret.

Reply:
Ah, of course. updated.